products-vertical.html

```diff
@@ -0,0 +1,210 @@
+                        <span class="check-group"><input type="checkbox" name="category"></input></input><label>Photography</label></span>
+                        <span class="check-group"><input type="checkbox" name="category"></input></input><label>Water</label></span>
+                      </div>
+                      <div class="rating-container flex-col-wrap-start">
+                        <h2>Rating</h2>
+                        <span class="check-group"><input type="radio" name="rating"></input><label>4 Stars & above</label></span>
+                        <span class="check-group"><input type="radio" name="rating"></input><label>3 Stars & above</label></span>
+                        <span class="check-group"><input type="radio" name="rating"></input><label>2 Stars & above</label></span>
+                        <span class="check-group"><input type="radio" name="rating"></input><label>1 Stars & above</label></span>
+                      </div>
+                      <div class="sortby-container flex-col-wrap-start">
+                        <h2>Sort by</h2>
+                        <span class="check-group"><input type="radio" name="sort-by"></input><label>Price - Low to High</label></span>
+                        <span class="check-group"><input type="radio" name="sort-by"><label>Price - High to Low</label></input></span>
+                      </div>
+                    </div>
+                </aside>
+                <main>
+                    <div class="main-section">
+                        <div class="main-container flex-row-wrap-center">
+                            <div class="card-container card-icon-overlay-container">
+                                <span class="card-wish-icon"><i class="fas fa-heart icon-unchecked"></i></span>
+                                <div class="card-body card-vertical-body">
+                                    <img class="card-img card-vertical-img" src="./assets/avatar-2.jpeg" sizes="(min-width: 600px) 200px,100px"/>
+                                    <div class="card-content-aside">
+                                        <div class="card-subtitle card-subtitle-lg">Picaso Painting</div>
+                                        <div class="card-title">$2000</div>
+                                    </div>
+                                </div>
+                                <div class="card-footer">
+                                   <button class="btn btn-primary">Add to Cart</button>
+                                </div>
+                            </div>
+                            <div class="card-container card-icon-overlay-container">
+                                <span class="card-wish-icon"><i class="fas fa-heart icon-unchecked"></i></span>
+                                <div class="card-body card-vertical-body">
+                                    <img class="card-img card-vertical-img" src="./assets/avatar-2.jpeg" sizes="(min-width: 600px) 200px,100px"/>
```

Comment:
same here in img tag

Reply:
Thanks for pointing out. I have added the alt tags.